cmd/fyne/commands/package.go

```diff
@@ -15,15 +14,84 @@ import (
 	"fyne.io/fyne/v2/cmd/fyne/internal/util"
 
 	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
 )
 
 const (
 	defaultAppBuild   = 1
 	defaultAppVersion = "1.0.0"
 )
 
-// Declare conformity to Command interface
-var _ Command = (*packager)(nil)
+// Package returns the cli command for packaging fyne applications
+func Package() *cli.Command {
+	p := &packager{}
+
+	return &cli.Command{
+		Name:        "package",
+		Usage:       "Packages an application for distribution.",
+		Description: "You may specify the -executable to package, otherwise -sourceDir will be built.",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "target",
+				Aliases:     []string{"os"},
+				Usage:       "The mobile platform to target (android, android/arm, android/arm64, android/amd64, android/386, ios).",
+				Destination: &p.os,
+			},
+			&cli.StringFlag{
+				Name:        "executable",
+				Aliases:     []string{"exe"},
+				Usage:       "The path to the executable, default is the current dir main binary",
+				Destination: &p.exe,
+			},
+			&cli.StringFlag{
+				Name:        "name",
+				Usage:       "The name of the application, default is the executable file name",
+				Destination: &p.name,
+			},
+			&cli.StringFlag{
+				Name:        "tags",
+				Usage:       "A comma-separated list of build tags.",
+				Destination: &p.tags,
+			},
+			&cli.StringFlag{
+				Name:        "appVersion",
+				Usage:       "Version number in the form x, x.y or x.y.z semantic version",
+				Destination: &p.appVersion,
+			},
+			&cli.IntFlag{
+				Name:        "appBuild",
+				Usage:       "Build number, should be greater than 0 and incremented for each build",
+				Destination: &p.appBuild,
+			},
+			&cli.StringFlag{
+				Name:        "sourceDir",
+				Aliases:     []string{"src"},
+				Usage:       "The directory to package, if executable is not set.",
+				Destination: &p.srcDir,
+			},
+			&cli.StringFlag{
+				Name:        "icon",
+				Usage:       "The name of the application icon file.",
+				Value:       "Icon.png",
+				Destination: &p.icon,
+			},
+			&cli.StringFlag{
+				Name:        "appiD",
+				Aliases:     []string{"id"},
+				Usage:       "For ios or darwin targets an appID is required, for ios this must match a valid provisioning profile",
```

Comment:
```suggestion
				Usage:       "For most platforms an appID in the form of a reversed domain names is required, for ios this must match a valid provisioning profile",
```

* perhaps I'm entirely wrong about iOS package names, please ignore this if that is the case.

Android specifically (because Java) used package names in the form of reversed domain names.

Microsoft also (on the Store) recommends package names following a similar form.